api/v1alpha1/validatorconfig_types.go

```diff
@@ -42,24 +45,21 @@ type Sink struct {
 
 // HelmRelease defines the configuration for a Helm chart release.
 type HelmRelease struct {
-	// Chart defines the Helm chart to be installed.
-	Chart HelmChart `json:"chart" yaml:"chart"`
+	// Name of the Helm chart.
+	Name string `json:"name" yaml:"name"`
+
+	// Version of the Helm chart.
+	Version string `json:"version" yaml:"version"`
 
 	// Values defines the values to be passed to the Helm chart.
 	Values string `json:"values" yaml:"values"`
 }
 
-// HelmChart defines the configuration for a Helm chart.
-type HelmChart struct {
-	// Name of the Helm chart.
-	Name string `json:"name" yaml:"name"`
-
+// HelmConfig defines the configuration for a Helm repository.
```

Comment:
Repository refers to the name of a chart within a _registry_. So this should say "for a Helm registry".